### PR TITLE
Upgrade systemd to 241

### PR DESCRIPTION
**Pull request by dm0-**
Depends on coreos/portage-stable#678

### CODE REVIEW

File: profiles/coreos/targets/generic/make.defaults [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2012 The Chromium OS Authors. All rights reserved.
 # Distributed under the terms of the GNU General Public License v2
 
-USE="cros-debug acpi usb symlink-usr usrmerge cryptsetup policykit"
```
Comment (cgwalters) [inline profiles/coreos/targets/generic/make.defaults]: Is that now the default?

Comment (dm0-) [reply profiles/coreos/targets/generic/make.defaults]: No, `usrmerge` was replaced by `split-usr` upstream, which has an inverted meaning.  https://github.com/gentoo/gentoo/commit/f602af077d978ba8982ced38021ab97e90b7fcca  Our systemd ebuild was the last place using `usrmerge`, so we can just drop it.

**Comment by ajeddeloh** [comment]
Why the one commit for sync+bump instead of the normal 2?

**Comment by dm0-** [comment]
No particular reason other than this package didn't use the two separate commits previously.  We can start doing that in the future, or I can rewrite this if you think it's better to start now.

**Comment by ajeddeloh** [comment]
/shrug I think it'd be good in the future, but if we haven't done it in the past there's no need to rewrite this now.

**Comment by dm0-** [comment]
Random dbus failures were encountered in test runs with this update; do not merge until it's resolved.

**Comment by dm0-** [comment]
I've bumped the commit to include coreos/systemd#104.  We discussed waiting to merge until July 31st to skip the next stable cycle.

**Comment by dm0-** [comment]
This is currently blocked on systemd/systemd#9656.

**Pull request merged**